runtime/src/accounts.rs

```diff
@@ -211,7 +211,7 @@ impl Accounts {
                     } else {
                         let (account, rent) = self
                             .accounts_db
-                            .load(ancestors, key)
+                            .load(ancestors, key, true)
```

Comment:
nits: it'll be hard to know the meaning of `true` here in the future. how about defining small wrappers like `load_on_chain`/`load_off_chain`? (well, i couldn't come up with good name...)

Reply:
I renamed the `is_transaction_load` to `is_root_fixed` and renamed these functions to `load_slow/load_slow_with_fixed_root` communicating whether the root can move while the load is happening, which should be the most important bit